src/aktualizr_repo/image_repo.h

```diff
@@ -11,6 +11,8 @@ class ImageRepo : public Repo {
                 const Delegation &delegation = {});
   void addCustomImage(const std::string &name, const Uptane::Hash &hash, uint64_t length, const Delegation &delegation);
   void addDelegation(const Uptane::Role &name, const std::string &path, KeyType key_type, bool terminating);
+  void revokeDelegtion(const Uptane::Role &name);
```

Comment:
There is a "a" missing in the method's name.

Reply:
Thanks